src/Monitoring/Monitoring.ArcadeServices/datasource/Production/AI-status.datasource.json

```
@@ -0,0 +1,22 @@
+{
+  "type": "grafana-azure-monitor-datasource",
+  "typeLogoUrl": "",
+  "access": "proxy",
+  "password": "",
+  "user": "",
+  "database": "",
+  "basicAuth": false,
+  "basicAuthUser": "",
+  "basicAuthPassword": "",
+  "withCredentials": false,
+  "isDefault": false,
+  "jsonData": {
+    "appInsightsAppId": "efa5f367-9edf-42d0-9cab-f0e27fa519c7",
+    "azureLogAnalyticsSameAs": false,
+    "cloudName": "azuremonitor"
+  },
+  "readOnly": false,
+  "secureJsonData": {
+    "appInsightsApiKey": "[vault(status-app-insights-api-key)]"
```

```suggestion
    "appInsightsApiKey": "[vault(dotneteng-status-app-insights-api-key)]"
```
Same idea here. Better to name it according to the AI instance it belongs to so we can track it down easily in the future. This also makes it consistent with the other API keys in the vault.